blog/2023-04-22-waking-up-from-hibernation/index.md

```diff
@@ -0,0 +1,108 @@
+---
+posttype: "blog"
+title: "Waking up from Hibernation"
+cover: "./cover.jpg"
+description: "Join me in this personal experience report on how I dust off and update my Terasology workspaces."
+tags: ["IMHO", "Comment"]
+author: "Skaldarnar"
+date: "2023-04-22"
+---
+
+_This is a personal experience report, sharing thoughts and insights into my workflow._
+
+It has been some time since our last update, and it also has been some time since my last contribution to Terasology.
+We had these quiet periods before, where everyone is charging up their batteries, learning new things, or just taking a break. 
+
+Coming back after such a long time can be hard - even if it seems quiet on a first glance, a lot of things might still have happened.
+And indeed: There have been hundreds of commits spread over our repositories since the [last blog post](/blog/tera-spotlight-gsoc-2021-final-show-tell/) in September 2021. 
+We had two releases since then, with [v5.3.0 / Alpha 23](https://github.com/MovingBlocks/Terasology/releases/tag/v5.3.0) being the latest one.
+
+Let me share how I updated my various workspaces when coming back after some time of absence...
+
+## Updating my Workspaces
+
+To start fresh I wanted to reset my workspaces to a clean state - discard any uncommitted changes (at this point I don't have any clue what they might have been about anyways) and pull the latest changes from upstream.
+It's not a hard task and I've done it uncountable times before, but not everybody might be that familiar with the tools and little helpers available to assist with that task.
+
+### Updating my Iota Workspace
+
+To bring my Iota workspace up to speed I'm used the `groovyw' utility tool that is shipped with the engine repository.
+You can run it easily via the provided wrapper scripts, that is `./groovyw` on Linux or `./groovyw.bat` on Windows.
+It comes in handy to initialize a workspace with a specific set of modules, fetch additional modules, or update everything via git.
+Our [Contributor Quick Start Guide](https://github.com/MovingBlocks/Terasology/wiki/Contributor-Quick-Start#set-up-your-terasology-development-workspace) also references it for the initial setup.
+
+But today I'm only interested in updating all my modules in the workspace.
+
+```
+groovyw module update-all
+```
```

Comment:
We might want to create an issue to improve formatting here:
![image](https://user-images.githubusercontent.com/29981695/233702317-c6c0d191-68d9-4660-a254-ffe8b38f69ca.png)

Reply:
See #211